<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查询读者信息表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查询图书信息表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查询借阅信息表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
